manifests/kustomize/README.md

```diff
@@ -6,8 +6,8 @@ This folder contains Kubeflow Pipelines Kustomize manifests for a light weight d
 
 Deploy latest version of Kubeflow Pipelines
 ```
-export PIPELINE_VERSION=0.1.31
-kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/namespaced-install.yaml
+export PIPELINE_VERSION=0.1.34
+for i in {1..2}; do kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/namespaced-install.yaml && break || sleep 1; done
```

Comment:
Can you help me understand why we need these retries in README?

Reply:
Besides, shall we sync this file with documentation on kubeflow/website?

Reply:
adding crd definition is a eventual consistent behavior. by adding crd and cr in the same command, somehow master always throw me 
```
error: unable to recognize "STDIN": no matches for kind "Application" in version "app.k8s.io/v1beta1"
```
which is really pain. blind retry works so i added it here.

Reply:
And regarding the doc, i can update once this is merged.

Reply:
> It might take a few seconds for the endpoint to be created. You can watch the Established condition of your CustomResourceDefinition to be true or watch the discovery information of the API server for your resource to show up.
https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition

I did some investigation, as you said, applying a CRD is eventual consistency (it takes some time before the k8s api endpoint is ready). So in reality it could take any amount of time before CRDs are ready. Using a retry here may work consistently now, but I think that's related to the time waited during retry just happen to be long enough for the CRD to be established. Any of these conditions may change in the future, e.g. if applying happens faster, or if CRDs take longer to establish, then this retry will break.

Also, everyone who read this the first time would wonder why we need a retry here.

I prefer we make the waiting explicit like the following. It is also pretty self explanatory about what we are doing.
```
kubectl apply -f crds
kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
kubectl apply -k envs/dev
```
https://stackoverflow.com/a/57126479